api/src/main/java/org/pmiops/workbench/api/CdrVersionsController.java

```diff
@@ -1,67 +1,29 @@
 package org.pmiops.workbench.api;
 
-import java.util.List;
-import java.util.logging.Logger;
-import java.util.stream.Collectors;
-import javax.inject.Provider;
-import org.pmiops.workbench.cdr.CdrVersionMapper;
 import org.pmiops.workbench.cdr.CdrVersionService;
-import org.pmiops.workbench.db.model.DbCdrVersion;
-import org.pmiops.workbench.db.model.DbUser;
-import org.pmiops.workbench.exceptions.ForbiddenException;
 import org.pmiops.workbench.model.CdrVersionListResponse;
-import org.pmiops.workbench.model.DataAccessLevel;
+import org.pmiops.workbench.model.CdrVersionMapResponse;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class CdrVersionsController implements CdrVersionsApiDelegate {
-  private static final Logger log = Logger.getLogger(CdrVersionsController.class.getName());
-
   private final CdrVersionService cdrVersionService;
-  private final CdrVersionMapper cdrVersionMapper;
-  private Provider<DbUser> userProvider;
 
   @Autowired
-  CdrVersionsController(
-      CdrVersionService cdrVersionService,
-      CdrVersionMapper cdrVersionMapper,
-      Provider<DbUser> userProvider) {
+  CdrVersionsController(CdrVersionService cdrVersionService) {
     this.cdrVersionService = cdrVersionService;
-    this.cdrVersionMapper = cdrVersionMapper;
-    this.userProvider = userProvider;
   }
 
   @Override
+  @Deprecated
   public ResponseEntity<CdrVersionListResponse> getCdrVersions() {
-    // TODO: Consider filtering this based on what is currently instantiated as a data source. Newly
```

Comment:
pushed this logic down to CdrVersionService.getCdrVersions() / getCdrVersionsbyTier() along with substantial changes